net/usrsock/usrsock_recvmsg.c

```diff
@@ -442,6 +442,12 @@ ssize_t usrsock_recvmsg(FAR struct socket *psock, FAR struct msghdr *msg,
       *fromlen = outaddrlen;
     }
 
+  if (conn->flags & USRSOCK_EVENT_REMOTE_CLOSED &&
```

Comment:
With this patch, recvmsg always return 0 even in the abnormal case happen(cable unplug or disconnect from AP), which may not good. We would prefer fix the test case here instead:
https://github.com/apache/incubator-nuttx-apps/pull/1304/commits/afd6554bab2acbf2cba4f20d39c46f36396a66fe

Reply:
@xiaoxiang781216 @anchao 
Should we revert this PR and fix it again?

Reply:
Yes, we can simply revert this patch. Let https://github.com/apache/incubator-nuttx-apps/commit/afd6554bab2acbf2cba4f20d39c46f36396a66fe fix it.